.eslintrc.json

```diff
@@ -6,16 +6,10 @@
   "rules": {
     "sort-imports": "error",
     "unicorn/prefer-module": "off",
-    "unicorn/import-style": [
-      "error",
-      {
-        "styles": {
-          "chalk": false
-        }
-      }
-    ],
+    "unicorn/import-style": "error",
     "unicorn/no-array-reduce": "off",
     "unicorn/prefer-array-some": "off",
-    "no-useless-constructor": "off"
+    "no-useless-constructor": "off",
+    "unicorn/prefer-string-replace-all": "off"
```

Comment:
I prefer replace-all for strings (over having to use regex for everything) and it's available in node16+